website_snippet_timeline/templates/snippets.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 Onestein
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <template id="snippets" inherit_id="website.snippets">
+        <xpath expr="//div[@id='snippet_feature']/div[hasclass('o_panel_body')]" position="inside">
+            <t t-snippet="website_snippet_timeline.website_snippet_timeline"
+               t-thumbnail="/website_snippet_timeline/static/src/img/snippet.png" />
+
+            <t t-snippet="website_snippet_timeline.website_snippet_timeline_item"
+               t-thumbnail="/website_snippet_timeline/static/src/img/snippet-item.png" />
+        </xpath>
+    </template>
+
+    <template id="snippet_options" inherit_id="website.snippet_options">
+        <xpath expr="." position="inside">
+            <div data-selector=".snippet_timeline_item"
+                 data-js="timeline_item"
+                 data-drop-in=".snippet_timeline ul" data-drop-near=".snippet_timeline ul > li">
+            </div>
+
+            <div data-js="margin-y"
+                 data-selector=".snippet_timeline_item">
+            </div>
+        </xpath>
+    </template>
+
+    <template id="website_snippet_timeline_item" name="Timeline - Item">
+        <li class="snippet_timeline_item mb64 o_animable">
```

Comment:
Put this inside of a section tag in order to avoid:
![image](https://user-images.githubusercontent.com/10233975/57183947-71bfbc80-6e79-11e9-94d3-891f76093080.png)

Reply:
fot get it, it's ok

I confirm the red is not related to my last coment, is because the snipet dones not fine any ul element to append on it